src/types/Backoffice.ts

```diff
@@ -0,0 +1,26 @@
+export type Category = 'Agenda' | 'Minutes' | 'Calendar';
+
+export interface Document {
+  id: string;
+  title: string;
+  url: string;
+  category: Category;
+  date?: string | null;
+  is_archived?: boolean;
+  fy: string;
```

Comment:
this fy type doesn't match the fy type underneath in GetDocsOptions
Please change to: fy?: string | null